integration/ats.sh.in

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+##############################################################################
+# Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+# other Serac Project Developers. See the top-level LICENSE file for
+# details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+##############################################################################
+
+########################
+## Test Invocations
+#
+# TOSS3: salloc -N2 ./ats.sh
+# BlueOS: lalloc 2 ./ats.sh
+# WSL: ./ats.sh
+
+# Variables used inside of the base test.ats
+export ATS_SERAC_BIN_DIR=@SERAC_BIN_DIR@/bin
+export ATS_SERAC_REPO_DIR=@SERAC_REPO_DIR@
+
+export ATS_BIN_DIR=@ATS_DIR@/bin
```

Comment:
I thought it was a little confusing that `ATS_DIR` gets an added `bin` here instead of being the full directory. Maybe we should make a note of that in the docs?

Reply:
Yea I tripped on this myself to be honest.  I was having it match the CMake variable.  I'll make note of it in the docs.

Reply:
I added a bunch of error checks that will yell at you if you get the wrong path.

Reply:
reworked the whole thing.. also moved this file back to the main repo.